xml/System.DirectoryServices.AccountManagement/PrincipalValueCollection`1.xml

```diff
@@ -656,7 +682,7 @@
       <Docs>
         <param name="value">An object to be added to the collection.</param>
         <summary>Adds the specified object to the end of the collection.</summary>
-        <returns>To be added.</returns>
+        <returns>The position into which the new element was inserted, or -1 to indicate that the item was not inserted into the collection.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Just curious why the `explicit interface member implementation` remark not added here

Reply:
@carlossanlop

Reply:
Probably a bug. I'll investigate. In the meanwhile, I can manually apply them:

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[
## Remarks
This member is an explicit interface member implementation. It can be used only when the <xref:System.DirectoryServices.AccountManagement.PrincipalValueCollection`1> instance is cast to an <xref:System.Collections.IList> interface.
          ]]></format>
        </remarks>
```